src/main/java/com/google/sps/model/GmailClient.java

```diff
@@ -70,6 +71,17 @@
     }
   }
 
+  /**
+   * Given some search queries, combine them into a larger query Will not remove
+   * duplicates/contradictions. TODO: Refactor query creation using the Builder pattern (Issue #100)
+   *
+   * @param queries gmail search queries
+   * @return String with queries space separated for use in Gmail
+   */
+  static String combineSearchQueries(String... queries) {
```

Comment:
Make it clear that you are effectively combining the queries using AND operators.

Reply:
Yep agreed. Thanks!